sql/ir_generator.go

```diff
@@ -142,13 +142,30 @@ func parseExpression(e interface{}) (interface{}, error) {
 	return nil, fmt.Errorf("not supported expr: %v", el)
 }
 
+func inferStringValue(expr string) interface{} {
+	ret, err := strconv.Atoi(expr)
+	if err == nil {
+		return ret
+	}
+	retFloat, err := strconv.ParseFloat(expr, 32)
+	if err == nil {
+		// NOTE: always use float32 for attributes, we may never use a
+		//       float64 value as some attribute.
+		return float32(retFloat)
+	}
+	retString := strings.Trim(expr, "\"")
+	retString = strings.Trim(retString, "'")
+	return retString
+}
+
 func parseFeatureColumn(el *exprlist) (codegen.FeatureColumn, error) {
 	head := (*el)[0].val
 	if head == "" {
 		return nil, fmt.Errorf("column description expects format like NUMERIC(key) etc, got %v", el)
 	}
 
 	switch strings.ToUpper(head) {
+	// TODO(typhoonzero): support FieldMeta configurations in column clause
```

Comment:
I don't quite understand. How do we support FieldMeta configurations in column clause?

Reply:
Updated comment to:

```
// TODO(typhoonzero): support FieldMeta configurations (DENSE/SPARSE) in column clause, e.g.
// CATEGORY_ID(DENSE(...))
```